README.md

```diff
@@ -1,16 +1,35 @@
 # Jade Service Runner UI
 
 <p align="center">
-<img height="256" width="256" src="https://github.com/etclabscore/jade-media-assets/raw/master/jade-logo-light/jade-logo-light%20(PNG)/256x256.png" />
+<img height="256" width="256" src="https://github.com/etclabscore/jade-media-assets/blob/master/jsr-gui/jsr-gui%20(PNG)/256x256.png?raw=true" />
```

Comment:
just a nitpick: we should name the asset dir `ui` to just be consistent

Reply:
fixed this with following commit